WeCantSpell.Hunspell.Tests/WeCantSpell.Hunspell.Tests.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;netcoreapp3.1;net462</TargetFrameworks>
```

Comment:
I think this bit here makes a (very quiet) statement about the versions where I am expecting this to be used.

Reply:
For context, this was previously `netcoreapp2.1` but ... MSFT has dropped support.